app/src/main/java/space/taran/arknavigator/mvp/model/repo/preview/generator/TxtPreviewGenerator.kt

```diff
@@ -0,0 +1,58 @@
+package space.taran.arknavigator.mvp.model.repo.preview.generator
+
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.text.Layout
+import android.text.StaticLayout
+import android.text.TextPaint
+import java.io.FileReader
+import java.nio.file.Path
+import space.taran.arknavigator.ui.App
+
+object TxtPreviewGenerator {
+    // it is padding in preview image
+    val padding = 10f * App.instance.resources.displayMetrics.density
```

Comment:
Do we need this for thumbnails too?

Reply:
Yes we need this for thumbnail. It is to add padding around the text in image.